src/sentry/runner/commands/import_system_symbols.py

```diff
@@ -0,0 +1,134 @@
+"""
+sentry.runner.commands.import_system_symbols
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2015 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import uuid
+import json
+import click
+import zipfile
```

Comment:
We should lazily import this inside `import_system_symbols`. We don't use zipfiles anywhere else, so no need to have it loaded in memory at all ties.